views/getinvolved.tmpl

```diff
@@ -66,11 +66,14 @@
 </div>
 <div class="container-fluid container-padded  bg-third">
   <div class="container">
-    <div class="row">
       <h2>How Do I Join?</h2>
-      <p>If you're ready to sign up right now, you can fill in the form below to sign up to our mailing list and computer system. It's instant and of course there's no commitment to do anything.</p>
-      <p>You can also come and <a href="/contact/" title="Contact us page">visit our station</a> in Vanbrugh College, where there'll almost always be someone there to talk to you and get you set up.</p>
-    </div>
+      <p>If you’re ready to join URY right now, we have some very easy steps to get you started...</p>
+      <ul>
+        <li><strong>Pay Membership - </strong>The easiest way to pay membership is through the <a href="//yusu.org/groups/university-radio-york/" target="_blank">yusu.org</a> website.</li>
+        <li><strong>Sign Up To MyRadio - </strong>MyRadio is our computer system for managing shows, members, and our mailing list. You can join MyRadio by completing the form below, it’s instant and of course there’s no commitment to do anything.</li>
+        <li><strong>Join Our Slack Team - </strong>Use slack; it's like Facebook Messenger but for URY team communication. Slack has the advantage of having a “channel” for each team, so you can join any channel and see what is happening. Our Slack team: <a href="//ury.slack.com" title="Join Our Slack Team" target="_blank">ury.slack.com</a> sign up using your university email address.</li>
```

Comment:
Okay, I only suggested it because of the use of strong as a intro to all of the list elements.